e2e/testcases/multi_sync_test.go

```diff
@@ -805,35 +805,21 @@ func TestControllerValidationErrors(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.MultiRepos)
 
 	testNamespace := k8sobjects.NamespaceObject(testNs)
-	if err := nt.KubeClient.Create(testNamespace); err != nil {
-		nt.T.Fatal(err)
-	}
+	nt.Must(nt.KubeClient.Create(testNamespace))
 	t.Cleanup(func() {
-		if err := nomostest.DeleteObjectsAndWait(nt, testNamespace); err != nil {
-			nt.T.Fatal(err)
-		}
+		nt.Must(nomostest.DeleteObjectsAndWait(nt, testNamespace))
 	})
 
 	nt.T.Logf("Validate RootSync can only exist in the config-management-system namespace")
-	rootSync := &v1beta1.RootSync{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "rs-test",
-			Namespace: testNs,
-		},
-		Spec: v1beta1.RootSyncSpec{
-			Git: &v1beta1.Git{
-				Auth: "none",
-			},
-		},
-	}
+	rootSync := k8sobjects.RootSyncObjectV1Beta1("rs-test")
+	rootSync.Namespace = testNs
```

Comment:
super nit: can use `core.Namespace` for oneline

Reply:
done